translations/pl.json

```diff
@@ -36,57 +36,57 @@
+	"Add machine IP to NO_PROXY environment variable": "Dodaj IP serwera do zmiennej środowiskowej NO_PROXY",
+	"Add, delete, or push a local image into minikube": "Dodaj, usuń lub wypchnij lokalny obraz do minikube",
+	"Add, remove, or list additional nodes": "Dodaj, usuń lub wylistuj pozostałe węzły",
+	"Adding node {{.name}} to cluster {{.cluster}}": "Dodawanie węzła {{.name}} do klastra {{.cluster}}",
 	"Additional help topics": "Dodatkowe tematy pomocy",
 	"Additional mount options, such as cache=fscache": "Dodatkowe opcje montowania, jak na przykład cache=fscache",
-	"Adds a node to the given cluster config, and starts it.": "",
-	"Adds a node to the given cluster.": "",
+	"Adds a node to the given cluster config, and starts it.": "Dodaje węzeł do konfiguracji danego klastra i wystartowuje go",
+	"Adds a node to the given cluster.": "Dodaje węzeł do danego klastra",
 	"Advanced Commands:": "Zaawansowane komendy",
-	"After the addon is enabled, please run \"minikube tunnel\" and your ingress resources would be available at \"127.0.0.1\"": "",
+	"After the addon is enabled, please run \"minikube tunnel\" and your ingress resources would be available at \"127.0.0.1\"": "Po włączeniu addona wykonaj komendę \"minikube tunnel\". Twoje zasoby będą dostępne pod adresem \"127.0.0.1\"",
 	"Aliases": "Aliasy",
-	"All existing scheduled stops cancelled": "",
+	"All existing scheduled stops cancelled": "Wszystkie zaplanowane zatrzymania zostały anulowane",
 	"Allow user prompts for more information": "",
 	"Alternative image repository to pull docker images from. This can be used when you have limited access to gcr.io. Set it to \\\"auto\\\" to let minikube decide one for you. For Chinese mainland users, you may use local gcr.io mirrors such as registry.cn-hangzhou.aliyuncs.com/google_containers": "",
-	"Amount of RAM allocated to the minikube VM (format: \u003cnumber\u003e[\u003cunit\u003e], where unit = b, k, m or g)": "Ilość zarezerwowanej pamięci RAM dla maszyny wirtualnej minikube (format: \u003cnumber\u003e[\u003cunit\u003e], where unit = b, k, m or )",
-	"Amount of RAM allocated to the minikube VM (format: \u003cnumber\u003e[\u003cunit\u003e], where unit = b, k, m or g).": "Ilość zarezerwowanej pamięci RAM dla maszyny wirtualnej minikube (format: \u003cnumber\u003e[\u003cunit\u003e], where unit = b, k, m or )",
+	"Amount of RAM allocated to the minikube VM (format: \u003cnumber\u003e[\u003cunit\u003e], where unit = b, k, m or g)": "Ilość zarezerwowanej pamięci RAM dla maszyny wirtualnej minikube (format: \u003cnumber\u003e[\u003cunit\u003e], gdzie jednostka to = b, k, m lub g)",
+	"Amount of RAM allocated to the minikube VM (format: \u003cnumber\u003e[\u003cunit\u003e], where unit = b, k, m or g).": "Ilość zarezerwowanej pamięci RAM dla maszyny wirtualnej minikube (format: \u003cnumber\u003e[\u003cunit\u003e], gdzie jednostka to = b, k, m or g)",
```

Comment:
This line is a duplicate of the line above and no longer exists in minikube, please delete it